FAQ.md

```diff
@@ -69,8 +69,41 @@ Metrics Server was tested to run within clusters up to 5000 nodes with average p
 
 Default 60 seconds, can be changed using `metrics-resolution` flag. We are not recommending setting values below 15s, as this is the resolution of metrics calculated within Kubelet.
 
+## Known issues
+
+#### Problems with Calico NetworkPolicy deployment with Kubeadm
+
+After adding `--kubelet-insecure-tls` and `--kubelet-preferred-address-types=InternalIP` arg in metrics-server container in [metrics-server-deployment.yaml](deploy/kubernetes/metrics-server-deployment.yaml), 
+the following problems occur when performing `kubectl top nodes` operations.
```

Comment:
```suggestion
#### Network problems

Metrics server needs to contact all nodes in cluster to collect metrics. Problems with network would can be recognized by following symptoms:

When running `kubectl top nodes` we get partial information. For example results like:
```

Reply:
Same problems can happen even without `--kubelet-insecure-tls` and `--kubelet-preferred-address-types=InternalIP`

Reply:
agree